stories/components/Tabs/default.md

```diff
@@ -37,6 +37,15 @@ A class name can be provided.
 </TabList>
 ```
 
+### Full Width
+
+To set the width of tabs to take 100% of the width, use the `fullWidth`
+prop.
+
+```jsx
+<Tabs fullWidth>…</Tabs>
+```
+
```

Comment:
Should the `variant` prop be added to the docs as well for visibility?

Reply:
✅